certified-connectors/TrueDialog SMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1762 @@
+              "title": "Callback Timestamp"
+            },
+            "CallbackToken": {
+              "type": "string",
+              "description": "CallbackToken"
+            },
+            "CallbackType": {
+              "type": "integer",
+              "format": "int32",
+              "description": "Always 11 for Incoming Message Callback",
+              "title": "CallbackType",
+              "x-ms-visibility": "internal"
+            },
+            "CallbackURL": {
+              "type": "string",
+              "description": "URL this callback has been sent to",
+              "title": "CallbackURL",
+              "x-ms-visibility": "internal"
+            },
+            "TransactionId": {
+              "type": "string",
+              "description": "Unique token",
+              "title": "TransactionId"
+            }
+          }
+        }
+      },
+      "post": {
+        "responses": {
+          "201": {
+            "description": "Incoming messages callback created"
+          }
+        },
+        "summary": "Incoming Message Received",
+        "description": "This operation triggers a flow whenever end user texts in",
+        "operationId": "Incoming_SMS_Received",
+        "x-ms-trigger": "single",
+        "parameters": [
+          {
+            "name": "AccountId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Use Root Account ID in most cases to subscribe for notifications.",
+            "x-ms-summary": "Account ID for subscription"
+          },
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "CallbackType": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "CallbackType",
+                  "title": "CallbackType",
+                  "default": 11,
+                  "x-ms-visibility": "internal"
+                },
+                "URL": {
+                  "type": "string",
+                  "description": "URL",
+                  "title": "URL",
+                  "x-ms-notification-url": true,
+                  "x-ms-visibility": "internal"
+                },
+                "Active": {
+                  "type": "boolean",
+                  "description": "Active",
+                  "title": "",
```

Comment:
@TrueDialogDynamics Empty property values can be removed until unless you have a specific reason to keep them.